### PR TITLE
Syntaxis: Change varable const 'pi2' by 'halfPi'

### PR DESCRIPTION
**Pull request by kikejimenez**
to improve code legibility, sintaxis has been changed

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 const pi = Math.PI;
-const pi2 = pi / 2;
```
Comment (Cereceres) [inline index.js:4]: why this change?? Is necessary?

Comment (kikejimenez) [reply index.js]: I think it is.

**Review by Cereceres — APPROVED** [review]
